File: Module 2 - Loops (Iteration)/for loop/Nested Loop/Nested.py
""" Syntax:
for i in range(rows):
    for j in range(columns):
        statement
"""
# Example 1 – Print Coordinates
print("Example 1 -- Print Coordinates")
for i in range(1, 4):
    for j in range(1, 4):
        print("(",i,",",j,")")

# Example 2 – Rectangle Pattern
print("Example 2 -- Rectangle Pattern")
for i in range(3):
    for j in range(5):
        print("*", end="")
    print()

# Example 3 – Number Pattern
print("Example 3 -- Number Pattern")
for i in range(3):
    for j in range(1, 6):
        print(j, end="")
    print()

File: Module 2 - Loops (Iteration)/for loop/Nested Loop/Nested.cpp
/* Syntax:
for(int i = 0; i < rows; i++)
{
    for(int j = 0; j < columns; j++)
    {
        statement;
    }

*/

#include<iostream>
using namespace std;

int main(){
// Example 1 – Print Coordinates
cout <<"Example 1 -- Print Coordinates\n"<<endl;
for(int i = 1; i <= 3; i++)
{
    for(int j = 1; j <= 3; j++)
    {
        cout << "(" << i << "," << j << ")" << endl;
    }
}

// Example 2 – Rectangle Pattern
cout <<"\nExample 2 -- Rectangle Pattern\n"<<endl;
for(int i = 1; i <= 3; i++)
{
    for(int j = 1; j <= 5; j++)
    {
        cout << "*";
    }
    cout << endl;
}

// Example 3 – Number Pattern
cout <<"\nExample 3 -- Number Pattern\n"<<endl;
for(int i = 1; i <= 3; i++)
{
    for(int j = 1; j <= 5; j++)
    {
        cout << j;
    }
    cout << endl;
}

}

